ecosystem/python/sdk/aptos_sdk/__init__.py

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env python3
```

Comment:
This shouldn't be here. We don't run these in a shell. My previous comment meant to say : if you want to target these python files for python3, this is not right way. Shebang should only be used for the scripts that are meant to be executed in a shell.

Reply:
let's remove the shebangs unless there's a main.